mavros_extras/src/plugins/vision_speed_estimate.cpp

```diff
@@ -84,6 +84,7 @@ class VisionSpeedEstimatePlugin : public MavRosPlugin {
 	void send_vision_speed(const geometry_msgs::Vector3 &vel_enu, const ros::Time &stamp) {
 		Eigen::Vector3d vel_;
 		tf::vectorMsgToEigen(vel_enu, vel_);
+		//Transform from ENU to NED frame
 		auto vel = UAS::transform_frame_enu_ned(vel_);
```

Comment:
No change here.  The enu->ned transform is correct and that function call's API was unchanged.